data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+	description "The Remnant are interested in learning what advances humanity has made over the past centuries. Bring them samples of advanced technology."
+	source "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 30
+	on offer
+		"remnant taely" ++
+		conversation
+			`As you enter the spaceport, you notice someone across the plaza appears to have been waiting for you. A tall, solidly built woman in short sleaved coveralls stands up from where she was sitting at the information desk. With a few quick gestures at the person behind the counter, she heads towards you with a confident stride. As she approaches you note that the crowds of people moving around the plaza seem to staying out of her way as if she was somone important. In contrast, the tools at her belt and the wear marks on her clothing suggest someone who is more comfortable inside a ship access duct than behind a desk.`
+			`	Before you can decide if you want to avoid this person or not, she is standing in front of you, clearing her throat. "You are captain <first> <last> of the <ship>?" She asks in a lyrical chant.`
+			choice
+				`	"That's me."`
+					goto admit
+				`	"I don't know who that is."`
+			`	She looks at you with an unscrutable expression for a moment. "Trying to be inconspicuous is admirable. Being oblivious to the fact that you stand out like a star in empty space is not."`
+			label admit
+			`	"I have an opportunity for you, if you are interested in being helpful to the Remnant." When you express your interest she leads you over to a balcony overlooking the primary shipyards. As she leans on the railing a cold breeze makes you shiver, but she seems oblivious to it. "As a prefect among the Remnant, I am responsible for this," she gestures at the shipyards, where you can see a myriad of the Remnant's alien-looking ships in various stages of construction or repair. "The ships we have serve us well, but they are old designs. Only the Gull is comparatively new. We need new ideas, new inspiration." She pauses, focusing briefly on a small bay where several people appear to be pulling what looks like a supercapacitor out of a gascraft.`
+			choice
+				`	"So, what are you asking?"`
+					goto request
+				`	"What are they doing?"`
+			`	"Oh, those are apprentices," she trills with a touch of amusement. "They need to learn how to handle unusual tech. One of the first steps is to deal with old tech. Like salvaging an obslete supercapacitor out of that grascraft so it can be reprocessed into something useful, and replacing it with newer integrated energy storage." She glances at you in amusement. "You've flown the gascraft, you know how old that ship is: Primitive energy systems, antique 'hard-shell' shields that have to be manually raised and lowered..." she trails off into momentary silence. "Be glad that you got one that had already been refurbished for your visit to Nenia. At least it had modern engines and an updated sensor array."`
```

Comment:
"grascraft". Also, should "Gascraft" be capitalized, like "Gull"? There are various places in this PR where the ship class is lower case.

Reply:
Thanks! fixed.

In terms of capitalization, everywhere else in the Remnant missions (Including missions by MZ) gascraft is not capitalized unless it is in a mission title.